src/components/AutoComplete/AutoComplete.tsx

```
@@ -0,0 +1,207 @@
+                    if (e.key === "Enter" || e.key === " ") {
+                      e.preventDefault();
+                      handleSelect(item);
+                    }
+                  }}
+
+                  className={cn(
+                    "text-muted-foreground cursor-pointer px-3 py-2 text-sm transition-colors",
+                    highlightedIndex === index &&
+                      "bg-accent text-accent-foreground",
+                  )}
+                >
```

Biome (2.1.2)

[error] The HTML element with the interactive role "option" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "option".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)